test/bin/manage_hypervisor_config.sh

```diff
@@ -123,6 +126,13 @@ action_create() {
 
     # Start nginx web server
     "${TESTDIR}/bin/manage_webserver.sh" "start"
+
+    if ! sudo vgdisplay rhel; then
```

Comment:
As above, this might need to be part of `scenario.sh` script.

Reply:
It might be, but see other comment for nuance around cleanup - it would be asymmetrical.